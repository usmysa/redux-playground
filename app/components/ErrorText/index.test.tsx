import { renderWithProviders } from "@/__tests__";
import { ErrorText } from ".";

describe("Error Text", () => {
  it("renders correctly", () => {
    const { container } = renderWithProviders(<ErrorText />);

    expect(container).toMatchSnapshot();
  });
});
