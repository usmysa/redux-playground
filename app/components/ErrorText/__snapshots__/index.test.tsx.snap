// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Error Text > renders correctly 1`] = `
<div>
  <div
    class="styles_wrapper__1xqviej9"
    style="--styleVars_alignItems__1xqviej0: center; --styleVars_flexDirection__1xqviej1: column; --styleVars_gap__1xqviej2: 0px; --styleVars_justifyContent__1xqviej4: flex-start; --styleVars_marginTop__1xqviej5: 0px; --styleVars_padding__1xqviej6: 0px; --styleVars_width__1xqviej8: 80px;"
  >
    <div
      class="styles_label__car12y0"
    >
      Errors
    </div>
    <div
      class="styles_value__car12y1"
    >
      0
    </div>
  </div>
</div>
`;
